Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent none
+    stages {
+        stage('Parallel Stage') {
+            when {
+                branch 'master'
```

Comment:
Uh, so you are not running any tests on PRs?!

Reply:
:man_facepalming: copy paste.